chat-plugins/shop.js

```diff
@@ -461,8 +461,15 @@ exports.commands = {
 		var bannedSymbols = /[ +<>$%‽!★@&~#卐|A-z0-9]/;
 		if (target.match(bannedSymbols)) return this.sendReply('Sorry, but you cannot change your symbol to this for safety/stability reasons.');
 		user.getIdentity = function() {
-			if (this.muted) return '!' + this.name;
 			if (this.locked) return '‽' + this.name;
+			if (roomid) {
```

Comment:
Right, the function argument should be <code>roomid</code>. I'mma update this PR real quick.